pkg/drivers/kic/oci/network_create.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oci
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+)
+
+// DefaultSubnet subnet to be used on first cluster
+const defaultSubnetAddr = "192.168.39.0"
```

Comment:
I'm concerned that this may cause routing issues if a host is configured on this subnet. Can you confirm that minikube will works properly in that environment (in say, a VM)?

Reply:
Maybe name this "firstSubnetAddr"?

Reply:
yes if cant create network, it will continue as old way